### ISSUE TITLE
moving_sd returns nan with long periods of no movement

### ISSUE THREAD

**Issue by m-patterson-wearable** [PfizerRD/scikit-digital-health#106]
In wear detection, moving_sd is returning nan when there are long periods of no change in accelerometer data.

`import numpy as np
from skdh.utility import moving_sd

x = np.array([-0.75500415, -0.47241098, -0.44129589])
test_input = np.repeat([x], 4000000, axis=0)

m_sd = moving_sd(test_input, 115199, 28799, axis=0, return_previous=False)

print(f"# of nan values: {np.isnan(m_sd).sum()}")`

Expect to have 0 coming out of moving_sd when there is no change in accelerometer data. Seems to only happen when the input accelerometer data size is beyond a certain length. I could not replicate it with smaller data sizes and window / step values.

**Comment by LukasAdamowicz** [comment]
This should be fixed in Version 0.11.6.post1, which should be distributed and on PyPI and conda shortly, can you confirm if this is still present on that version?

**Comment by m-patterson-wearable** [comment]
yes, I can confirm I was using v 0.11.6post1. I git cloned the repo to get it, not from PyPi.
